lsp-haskell.el

```diff
@@ -1,7 +1,7 @@
 ;;; lsp-haskell.el --- Haskell support for lsp-mode
 
-;; Version: 1.0
-;; Package-Requires: ((emacs "24.3") (lsp-mode "3.0"))
+;; Version: 1.1
+;; Package-Requires: ((emacs "24.3") (lsp-mode "3.0") (haskell-mode "16.1"))
```

Comment:
16.1 appears to be current at the time of writing. I'm not really sure what to put here.